client/ayon_core/plugins/publish/collect_rendered_files.py

```diff
@@ -31,6 +31,9 @@ class CollectRenderedFiles(pyblish.api.ContextPlugin):
     # Keep "filesequence" for backwards compatibility of older jobs
     targets = ["filesequence", "farm"]
     label = "Collect rendered frames"
+    settings_category = "core"
+
+    remove_files = True
```

Comment:
See [here](https://github.com/ynput/ayon-core/pull/1214/files#r2047019732)
```suggestion
    remove_files = False
```